website/blog/2023-08-25-silver.md

```diff
@@ -0,0 +1,399 @@
+---
+author: Jakub Ciesluk
+title: Metals v1.0.1 - Silver
+authorURL: https://github.com/jkciesluk
+authorImageURL: https://github.com/jkciesluk.png
+---
+
+We're happy to announce the release of Metals v1.0.1, which comes with
+a couple of exciting new features and stability improvements.
+
+<table>
+<tbody>
+  <tr>
+    <td>Commits since last release</td>
+    <td align="center">82</td>
+  </tr>
+  <tr>
+    <td>Merged PRs</td>
+    <td align="center">74</td>
+  </tr>
+    <tr>
+    <td>Contributors</td>
+    <td align="center">12</td>
+  </tr>
+  <tr>
+    <td>Closed issues</td>
+    <td align="center">19</td>
+  </tr>
+  <tr>
+    <td>New features</td>
+    <td align="center">3</td>
+  </tr>
+</tbody>
+</table>
+
+For full details: [https://github.com/scalameta/metals/milestone/59?closed=1]
+(https://github.com/scalameta/metals/milestone/59?closed=1)
+
+Metals is a language server for Scala that works with VS Code, Vim, Emacs and
+Sublime Text. Metals is developed at the
+[Scala Center](https://scala.epfl.ch/) and [VirtusLab](https://virtuslab.com)
+with the help from [Lunatech](https://lunatech.com) along with contributors from
+the community.
+
+## TL;DR
+
+Check out [https://scalameta.org/metals/](https://scalameta.org/metals/), and
+give Metals a try!
+
+
+- [Completions in java files](#completions-in-java-files)
+- [Liveness monitor](#liveness-monitor)
+- [Improvements for enums](#improvements-for-enums)
+- [Semantic highlighting](#semantic-highlighting)
+- [Miscellaneous](#miscellaneous)
+
+
+## Completions in Java files
+
+Thanks to amazing work done by [VankaTaganai](https://github.com/VankaTaganai) and further improvements by [tgodzik](https://github.com/tgodzik),
+you will be able to get completions in Java files. It should make 
+working with Java sources easier, though Metals will not strive to be a full Java language server.
+
+![java-completion](https://i.imgur.com/wLlh7ig.gif)
+
+## Liveness monitor
+
+Up to the current release, users would not be informed if something happened
+and the build server stopped responding. Thanks to [kasiaMarek](https://github.com/kasiaMarek) Metals will now monitor the state 
+of the build server and notify the user if it becomes unresponsive.
```

Comment:
Maybe also write smth in lines of: "Dismissing the notification will dismiss it for the current connection".